scripts/generate_loitering_events

```diff
@@ -43,7 +45,7 @@ fi
 echo "Ensuring events table ${DEST_TABLE} exists"
 TABLE_DESC=(
   "* Pipeline: ${PIPELINE} ${PIPELINE_VERSION}"
-  "* Source: ${SOURCE_TABLE}"
+  "* Source: ${SOURCE_LOITERING}"
```

Comment:
If it is possible, could you please add the rest of sources here?
SOURCE_SEGMENT_INFO, SOURCE_SEGMENT_INFO, SOURCE_SPATIAL_MEASURES and SOURCE_RESEARCH_SEGS 
It's useful when you need to check the dependencies :)

Reply:
Every arg is logged in the line just before that `Source: ...` one. I think the `Source: ...` doesn't make much sense now that we log every single arg on this.